packages/lesswrong/server/databaseSettings.ts

```diff
@@ -8,12 +8,6 @@ import { ensureIndex } from '../lib/collectionIndexUtils';
 
 const runValidateSettings = false
 
-ensureIndex(DatabaseMetadata, {
```

Comment:
Is this redundant?  Seems unlikely to come up, but would be bad if someone accidentally inserted e.g. a duplicate `serverSettings` or something

Reply:
Yeah - there's an identical index defined using `ensureCustomPgIndex` in `collections/databaseMetadata/collection.ts`